sdk/trace/attributesmap.go

```diff
@@ -62,21 +61,23 @@ func (am *attributesMap) add(kv label.KeyValue) {
 	}
 }
 
-func (am *attributesMap) toSpanData(sd *trace.SpanData) {
-	len := am.evictList.Len()
-	if len == 0 {
-		return
+// toKeyValue copies the attributesMap into a slice of label.KeyValue and
+// returns it. If the map is empty, a zero-length slice is returned.
+// TODO: Is it more efficient to return a pointer to the slice?
+func (am *attributesMap) toKeyValue() []label.KeyValue {
+	var attributes []label.KeyValue
+
+	if am.evictList.Len() == 0 {
+		return attributes
 	}
```

Comment:
```suggestion
// toKeyValue copies the attributesMap into a slice of label.KeyValue and
// returns it. If the map is empty, a nil is returned.
// TODO: Is it more efficient to return a pointer to the slice?
func (am *attributesMap) toKeyValue() []label.KeyValue {
	if am.evictList.Len() == 0 {
		return nil
	}
	
	var attributes []label.KeyValue
```

Nit

Reply:
Thanks! I actually wasn't aware nil was the zero value of a slice. Fixed.